ChiamakaObitube/PropertyPro-Lite
# #167085396 Setup Dev Environment and Express Server

**What does this PR do?**
This PR sets up a server side development environment for PropertyPro-Lite.

**Description of Task to be completed?**
Have the following working: Port is running on localhost,

**How should this be manually tested?**
The following steps can be used:
Clone the repository, cd into the server folder and run npm install on the terminal and start the server with npm run dev. There should be a port running on localhost:3000 with welcome message to PropertyPro-Lite/

**Any background context you want to provide?**
User should ensure there is node, npm installed on their machine.

**What are the relevant pivotal tracker stories?**
[167085396](https://www.pivotaltracker.com/story/show/167085396)
[Delivers #167085396]
